lib/registration.pm

```diff
@@ -100,29 +100,29 @@ sub fill_in_registration_data {
                     type_string $regcode;
                     sleep 1;
                     save_screenshot;
-                    send_key 'alt-n';     # next
                 }
             }
+            send_key 'alt-n';    # next
```

Comment:
I wonder - you move the next out of the loop. What is ACKing the regcode for the addon so we can type the next? Do you have a link to a test run?